apps/global_preferences.h

```diff
@@ -30,13 +30,23 @@ class GlobalPreferences {
   void setTempExamMode(ExamMode examMode);
   bool showPopUp() const { return m_showPopUp; }
   void setShowPopUp(bool showPopUp) { m_showPopUp = showPopUp; }
+  bool dfuStatus() const {return m_dfuUnlocked;}
+  void setDfuStatus(bool status) {m_dfuUnlocked=status;}
+  int dfuCurrentStep() const {return m_dfuStep;}
+  void dfuIncreaseStep() {m_dfuStep++;}
+  void dfuResetStep() {m_dfuStep = 0;}
+  int getDfuLevel() const {return m_dfuProtectLevel;}
+  void setDfuLevel(int level) {m_dfuProtectLevel = level;}
+  bool showDfuDeacAlert() const {return m_showDeacAlert;}
+  void setDfuDeacAlert(bool value)  {m_showDeacAlert = value;}
```

Comment:
Maybe replace "DeacAlert" by "DisabledAlert" here and everywhere it is called in the code